### ISSUE TITLE
PHP Fatal error during platform check

### ISSUE THREAD

**Issue by Prometee** [dbrekelmans/browser-driver-installer#81]
Since v1.0.5 I encountered this error on MacOs and Docker (Linux alpine image)

```
PHP Fatal error:  Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.1.0". You are running 8.0.30. in phar:///..../.composer/vendor/dbrekelmans/bdi/bdi/vendor/composer/platform_check.php on line 24
```

Here is the detailed output :
```shell
# composer global exec bdi -vvv

> __exec_command: bdi
Executing command (CWD): bdi

Box Requirements Checker
========================

> Using PHP 8.0.30
> PHP is using the following php.ini file:
  /usr/local/etc/php/8.0/php.ini

> Checking Box requirements:
  ✔ The application requires a version matching "^7.2|^8.0".
  ✔ The application requires the extension "json".
  ✔ The package "myclabs/php-enum" requires the extension "json".
  ✔ The application requires the extension "zip".
  ✔ The application requires the extension "zlib".
  ✔ The package "symfony/service-contracts" conflicts with the extension "psr".
  
                                                                                                                                                                                                                                   
 [OK] Your system is ready to run the application.                                                                                                                                                                                 
                                                                                                                                                                                                                                   

PHP Fatal error:  Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.1.0". You are running 8.0.30. in phar:///Users/prometee/.composer/vendor/dbrekelmans/bdi/bdi/vendor/composer/platform_check.php on line 24

Fatal error: Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.1.0". You are running 8.0.30. in phar:///Users/prometee/.composer/vendor/dbrekelmans/bdi/bdi/vendor/composer/platform_check.php on line 24
Script bdi handling the __exec_command event returned with error code 255
```

Using the v1.0.4 no issue encountered.

**Comment by spipu** [comment]
It is due to this commit : https://github.com/dbrekelmans/browser-driver-installer/commit/6ac390c7cddaacc82118b2dac0cc3787a936a850
By building on PHP8.2 instead of PHP7.2, it increase the minimal allowed version of php in the platform_check.php file of the phar.

@dbrekelmans Because you test all on accept PHP7.2 on https://github.com/dbrekelmans/browser-driver-installer/blob/acf0efa4a79785a670a34af7433fdce3b3bf0f34/.github/workflows/qa.yml you must build also on php7.2 and not php8.2

**Comment by cafferata** [comment]
Yes, same as us https://github.com/dbrekelmans/browser-driver-installer/pull/78#issuecomment-1727597625

**Comment by dbrekelmans** [comment]
This should now be fixed in [1.0.7](https://github.com/dbrekelmans/bdi/releases/tag/1.0.7)

**Comment by Prometee** [comment]
I can confirm it's fixed, thank you guys
